lib/build-group-id.js

```diff
@@ -33,9 +34,11 @@ module.exports = function buildGroupId({
     }
   }
 
-  const cloakedMsgId = new CloakedMsgId(msgId, readKey).toString()
+  const cloakedMsgId = base64url.escape(
+    new CloakedMsgId(msgId, readKey).toString()
+  )
```

Comment:
cc @staltz can you speak this this? these are different functions but I don't know if it matters.
This module `base64url.escape` does: https://github.com/joaquimserafim/base64-url/blob/master/index.js#L10-L20
```js
function unescape (str) {
  return (str + '==='.slice((str.length + 3) % 4))
    .replace(/-/g, '+')
    .replace(/_/g, '/')
}

function escape (str) {
  return str.replace(/\+/g, '-')
    .replace(/\//g, '_')
    .replace(/=/g, '')
}
```
# :hot_pepper:  DOES THE `=` MATTER?

Meanwhile `ssb-uri2` does: https://github.com/ssbc/ssb-uri2/blob/main/src/index.ts#L33-L41
```js
const Base64 = {
  unsafeToSafe(input: string) {
    return input.replace(/\+/g, '-').replace(/\//g, '_');
  },

  safeToUnsafe(input: string) {
    return input.replace(/-/g, '+').replace(/_/g, '/');
  },
};
```

Reply:
I think these functions are functionally the same. the = is filler right

Reply:
yeah if you're already using ssb-uri2 you should expose that escape unescape function and use that